h3/src/quic.rs

```diff
@@ -96,6 +96,17 @@ pub trait RecvStream {
         cx: &mut task::Context<'_>,
     ) -> Poll<Result<Option<Self::Buf>, Self::Error>>;
 
+    /// Poll the stream for a precise number of bytes
+    ///
+    /// The `buf` slice should be filled with as much received data as possible. When
+    /// the receive side will no longer receive more data (such as because
+    /// the peer closed their sending side), this should return `None`.
+    fn poll_read(
```

Comment:
Why the inclusion of this new method, instead of using `poll_data`?

Reply:
`poll_data()` reads everything available, but [here](https://github.com/hyperium/h3/pull/47/files/cc0e3a0e21447b4f2e561b52d78d43adb9eee503#diff-0845c3b9e922cf769bbc50f9ecaf7abc30f9b57bfaa73b1d4fd4b13c758e7e91R103) we want to consume only the first `VarInt` of the new stream.